xtr/src/crate_visitor.rs

```diff
@@ -127,6 +127,18 @@ where
                 .unwrap_or_else(|err| self.mod_error = Some(err));
         }
 
+        if let Some(without_prefix) = self.current_path.to_str().unwrap().strip_prefix("src/") {
```

Comment:
i don't think you should strip the src prefix

Reply:
Indeed, I just found a way to avoid this stripping. We know that self.current_path contains abc/def/ghi/foo.rs, so if we strip it down to abc/def/ghi/foo and then try if abc/def/ghi/foo/bar.rs exists, that also works.

Regarding the CI failure, perhaps the action is also broken on master. At least I tried the patch with 1.65.0, and there your code builds fine. :-)

Please let me know if I should address anything else in the patch. Thanks.